### PR TITLE
✨ Add OpenJS Foundation infos to footer

### LINKED ISSUE #3892: Update legal and add OpenJS foundation information
As per https://github.com/ampproject/wg-outreach/issues/60.

The foundation's logo from [openjs-foundation/artwork](https://github.com/openjs-foundation/artwork) needs to be added to the footer. Additionally links to legal documents need to be updated to the following:

```html
<p>Copyright <a href="https://openjsf.org">OpenJS Foundation</a> and AMP Project contributors. All rights reserved. The <a href="https://openjsf.org">OpenJS Foundation</a> has registered trademarks and uses trademarks. For a list of trademarks of the <a href="https://openjsf.org">OpenJS Foundation</a>, please see our <a href="https://trademark-policy.openjsf.org">Trademark Policy</a> and <a href="https://trademark-list.openjsf.org">Trademark List</a>. Trademarks and logos not indicated on the <a href="https://trademark-list.openjsf.org">list of OpenJS Foundation trademarks</a> are trademarks&trade; or registered&reg; trademarks of their respective holders. Use of them does not imply any affiliation with or endorsement by them.</p>
<p><a href="https://openjsf.org">The OpenJS Foundation</a> | <a href="https://terms-of-use.openjsf.org">Terms of Use</a> | <a href="https://privacy-policy.openjsf.org">Privacy Policy</a> | <a href="https://bylaws.openjsf.org">OpenJS Foundation Bylaws</a> | <a href="https://trademark-policy.openjsf.org">Trademark Policy</a> | <a href="https://trademark-list.openjsf.org">Trademark List</a> | <a href="https://www.linuxfoundation.org/cookies">Cookie Policy</a></p>
```

### PR DESCRIPTION
**Pull request by matthiasrohmer**
Bildschirmfoto 2020-05-14 um 16 44 26

Resolves #3892.

### CODE REVIEW

**Comment by tobie** [comment]
Are you planning to address https://github.com/ampproject/wg-outreach/issues/60#issuecomment-624094267 separately?

**Comment by matthiasrohmer** [comment]
Good point. @sebastianbenz, did you already think about https://github.com/ampproject/wg-outreach/issues/60#issuecomment-624094267? I have no clue where this could go...

**Comment by sebastianbenz** [comment]
Sorry, didn't realize that this was blocked on my feedback. I'd add this at the end of the OpenJS trademark paragraph.

**Comment by matthiasrohmer** [comment]
Thanks! Added the following:

```html
The services available at <a href="https://cdn.ampproject.org">cdn.ampproject.org</a> are provided by Google. Their respective <a href="https://policies.google.com/privacy?hl=en">privacy policy</a> applies.
```

Opinions on the wording are welcome 🙂

**Comment by brianwarner** [comment]
The styling and placement look great to me.

**Comment by matthiasrohmer** [comment]
Thanks for the quick feedback, @brianwarner! Is there someone else who needs to approve, @tobie?

File: frontend/templates/views/partials/footer.j2 [hunk 1]
```diff
@@ -79,12 +86,20 @@
...
       </ul>
-      <div class="ap-o-footer-copyright">{{ _('&copy; 2020 AMP Open Source Project') }}</div>
     </div>
-
+    <div class="ap-o-footer-copyright">
+      {{ _('&copy; <a href="https://openjsf.org">OpenJS Foundation</a> and AMP Project contributors. All rights reserved. The <a href="https://openjsf.org">OpenJS Foundation</a> has registered trademarks and uses trademarks. For a list of trademarks of the <a href="https://openjsf.org">OpenJS Foundation</a>, please see our <a href="https://trademark-policy.openjsf.org">Trademark Policy</a> and <a href="https://trademark-list.openjsf.org">Trademark List</a>. Trademarks and logos not indicated on the <a href="https://trademark-list.openjsf.org">list of OpenJS Foundation trademarks</a> are trademarks&trade; or registered&reg; trademarks of their respective holders. Use of them does not imply any affiliation with or endorsement by them.') }}
+      <br><br>
+      {{ _('The services available at <a href="https://cdn.ampproject.org">cdn.ampproject.org</a> are provided by Google. Their respective <a href="https://policies.google.com/privacy?hl=en">privacy policy</a> applies.') }}
```
Comment (tobie) [inline frontend/templates/views/partials/footer.j2:101]:
```suggestion
      {{ _('The services available at <a href="https://cdn.ampproject.org">cdn.ampproject.org</a> are provided by Google and the respective <a href="https://policies.google.com/privacy?hl=en">privacy policy</a> applies.') }}
```

**Comment by matthiasrohmer** [comment]
Thanks! Regarding the timeline: there's no automated production deployment but only for staging. As soon as it looks good there I (or Sebastian, whoever is quicker 😉 ) can trigger a prod build.

**Pull request merged**